testing/opencolorio/APKBUILD

```diff
@@ -0,0 +1,77 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer:
```

Comment:
I must have misunderstood the purpose of the maintainer field. The simple example ( https://wiki.alpinelinux.org/wiki/APKBUILD_examples:Simple ) also contains a blank maintainer field.